d2bs/kolbot/libs/MuleLogger.js

```diff
@@ -67,17 +69,33 @@ var MuleLogger = {
 	},
 
 	inGameCheck: function () {
+		var	randloc,
+			tick = ((Math.random() * 30) + 90), // antiidle movement trigger
+			loc = ["stash", "waypoint", "portalspot", "akara", "charsi", "kashya", "cain", "gheed"];
```

Comment:
Move the anti-idle code to prototypes and make something like me.antiIdle(), then in your while loop you can do that. Also this can be used later in other scripts